site/source/docs/optimizing/Optimizing-Code.rst

```diff
@@ -44,9 +44,13 @@ Compiling source files to object files works as you'd expect in a native build s
 Link Times
 ==========
 
-To skip extra optimization work at link time, link with ``-O0`` or ``-O1``. It
-is ok to link with those flags even if the source files were compiled with a
-different optimization level.
+To skip extra optimization work at link time, link with ``-O0`` or ``-O1``. In
+those modes Emscripten focuses on faster iteration times.
+
+.. note::
```

Comment:
Is this what results in a really big blue box on the web page? I'm not sure it's worth such a large UI element. Maybe put it in parentheses?

Reply:
Sounds good, done.